pyproject.toml

```diff
@@ -3,17 +3,17 @@
 # "Prerequisites" in User's Guide,
 # build jobs in `.github/workflows/*.yml`.
 requires = [
-    "setuptools >= 61.0.0",
+    "setuptools >=61.0.0",
     "wheel",
-    "numpy < 2",
+    "numpy >=1.20.0",
     "packaging",
     "py-cpuinfo",
     "Cython >=3.0.10",
```

Comment:
... and arguably we shouldn't have bumped the `cython` build requirement either if we're trying to help packagers, so I'll revert that, too.

Reply:
Cython is not a big issue, for Ubuntu/Debian at least.
... but going back doesn't hurts for the time being